modules/nw-ingress-setting-select-subnet-loadbalancerservice.adoc

```diff
@@ -0,0 +1,92 @@
+// Modules included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-setting-select-subnet-Loadbalancerservice_{context}"]
+= Choosing subnets while creating a LoadBalancerService Ingress Controller
++
+You can manually specify load balancer subnets for Ingress Controllers in an existing cluster. By default, the load balancer subnets are automatically discovered by AWS, but specifying them in the Ingress Controller overrides this, allowing for manual control.
+
+.Prerequisites
+* You must have an installed AWS cluster.
+* You must know the names or IDs of the subnets to which you intend to map your `IngressController`.
+
+.Procedure
+
+. Create a custom resource (CR) file.
++
+Create a YAML file (e.g., `sample-ingress.yaml`) with the following content:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  namespace: openshift-ingress-operator
+  name: <name>
+spec:
+  domain: <domain>
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: External
+  dnsManagementPolicy: Managed
+----
+
+. Create a custom resource (CR) file.
++
+Add subnets to your YAML file:
++
+--
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name:  <name> <1>
+  namespace: openshift-ingress-operator
+spec:
+  domain: <domain> <2>
+  endpointPublishingStrategy:
+    type: LoadBalancerService
+    loadBalancer:
+      scope: External
+      providerParameters:
+        type: AWS
+        aws:
+          type: Classic
+          classicLoadBalancer: <3>
+            subnets:
+              ids: <4>
+              - <subnet> <5>
+              - <subnet>
+              - <subnet>
+dnsManagementPolicy: Managed
+----
+<1> Replace `<name>` with a name for the `IngressController`.
+<2> Replace `<domain>` with the DNS name serviced by the `IngressController`.
+<3> You can also use the `networkLoadBalancer` field if using an NLB.
+<4> You can optionally specify a subnet by name using the `names` field instead of specifying the subnet by ID.
+<5> Specify subnet IDs (or names if you using `names`).
++
+[IMPORTANT]
+====
+You can specify a maximum of one subnet per availability zone. Only provide public subnets for external Ingress Controllers and private subnets for internal Ingress Controllers.
+====
+--
++
+. Apply the CR file.
```

Comment:
This stray `+` might choke the build (or not, idk, but if you see a mysterious mess try removing it)
```suggestion
--

. Apply the CR file.
```